kolibri_explore_plugin/assets/src/components/AboutModal.vue

```diff
@@ -371,6 +433,173 @@
     },
     $trs: {
       privacyPolicyHeader: 'Privacy Policy',
+      aboutHeading: {
+        message: 'Endless Key — learning and discovery resources, no internet required.',
+        context: 'Heading at the top of the About dialogue',
+      },
+      aboutBody: {
+        message:
+          'The Endless Key unlocks the potential for student curiosity and choice through independent learning. Curated to be interest-based and inclusive, our collection of content encourages students to discover knowledge and skills that will support them now and in their futures. The Endless Key is designed to complement the broadband-for-all roll-out in the United States and provide offline students with equitable access to engaging content. Students can access Endless Key content by downloading the Endless Key app (available for ChromeOS and Windows). After the initial download, no internet is required to interact with the content.',
+        context: 'Paragraph of information about Endless Key',
+      },
+      releaseInformationHeading: {
+        message: 'Release information',
+        context: 'Heading for a table of software release information',
+      },
+      releaseKey: {
+        message: 'Release:',
+        context: 'Table key for software release information',
+      },
+      dateKey: {
+        message: 'Date:',
+        context: 'Table key for software release date',
+      },
+      commitKey: {
+        message: 'Commit:',
+        context: 'Table key for software release commit',
+      },
+      kolibriKey: {
+        message: 'Kolibri:',
+        context: 'Table key for software Kolibri version',
+      },
+      supportHeading: {
+        message: 'Support',
+        context: 'Heading for technical support section',
+      },
+      supportBody: {
+        message:
+          'You can find more information on how to use your Endless Key on <strong>The Quick Start Guide</strong> document. For more information, when you have an internet connection, you can access the Endless Support Site at <a href="https://support.endlessos.org/en/endless-key" target="_blank">https://support.endlessos.org/en/endless-key</a>.',
```

Comment:
I think it's fine to have the v-html. I considered using a name like `"...you can access the Endless Support Site at { siteURL}."` and have that URL as vue data. Maybe it's better to de-hardcode the URL from the translators message, but doesn't avoid the use of v-html.

Reply:
It would avoid the problem of a malicious translator sending users to a different URL, but what are the chances?

Reply:
Yeah, I think the use of `v-html` is fine as things are, but it’s something we want to be careful not to break in future.

Reply:
This is also something I think we could iterate on in future if it turns out that having the markup in the translatable string causes problems.

In GNOME, this kind of setup occasionally causes problems where translators literally translate `<strong>` to (for example) `<fuerte>`. But that is quite rare.